tensorflow_datasets/core/features/skip_encoding_feature.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow_datasets as tfds
+from tensorflow_datasets.core.features import feature
+
+
+class SkipEncodingFeature(feature.FeatureConnector):
```

Comment:
Also, it's important for documentation to be correct and so if we stick with this approach, we'll want to update how [`FeatureConnector`s display](https://github.com/tensorflow/datasets/blob/d2123d21ec68607a0de830ab3afcde36162190f0/tensorflow_datasets/core/features/feature.py#L283) so that `SkipEncodingFeature` could forward the base connector and not show up in documentation.

Reply:
Happy to make this change - possibly via a `DelegatingFeatureConnector` that redirects all base methods? I've done a lot of work in dart and really like the `DelegatingXXX` pattern... but I'm wary of throwing in too many micro-classes in a single request.